### PR TITLE
Fixed duplicate images in preview.

### PR DESCRIPTION
**Pull request by eozan**
SDPA-289

### CODE REVIEW

**Comment by pavlovicnemanja** [comment]
:+1:

File: scripts/superdesk-authoring/views/full-preview.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
             </div>
             <div class="headline">{{item.headline}}</div>
 
-            <div ng-repeat="association in item.associations" class="association">
+            <div ng-repeat="(associationName, association) in item.associations" class="association" ng-if="!isEmbedded(associationName)">
```
Comment (actionless) [inline scripts/superdesk-authoring/views/full-preview.html]: won't filter look more understandable here?

**Comment by ioanpocol** [comment]
:+1:

**Comment by eozan** [comment]
Changed to use filter as @actionless suggested.

**Pull request merged**